addons/sys_gui/XEH_postInit.sqf

```diff
@@ -3,6 +3,7 @@
 
 if (!hasInterface) exitWith {};
 
+
```

Comment:
Remove new line?

Reply:
Split into categories: Inventory, Volume Control and now removed Vehicle Info UI.